book/src/how_to/rust-std.md

```diff
@@ -0,0 +1,29 @@
+# Combining Rust `std` with `uefi`
+
+## TL;DR
+
+In Mid-2024, we recommend to stick to our normal guide. Use this document as
+guide and outlook for the future of UEFI and Rust.
+
+## About
+
+In the `uefi` crate, we provide tooling to create a `no_std` + `no_main`
```

Comment:
I'm still not 100% satisfied with the phrasing of this paragraph. Any idea?

Reply:
One thing that would help I think is to drop the "standard binary" terminology; stick to `no_std` and the `std` crate.

Here's my attempt:

---
Programs created with the `uefi` crate are typically created with `no_std` and `no_main`. A `no_std` crate can use the `core` and `alloc` parts of Rust's standard library, but not `std`. A `no_main` executable does not use the standard `main` entry point, and must define its own entry point; `uefi` provides the `entry` macro for this purpose.

Rust has added partial support for building UEFI executables without `no_std` and `no_main`. Some functionality requires a nightly toolchain, and some parts of `std` are gated by the unstable `uefi_std` feature. Follow the [tracking] issue for details.

[tracking]: https://github.com/rust-lang/rust/issues/100499